From c5caaf32a6ee2d25d7f83f46649a5caf1cc4fc16 Mon Sep 17 00:00:00 2001
From: Irina Tirdea <irina.tirdea@intel.com>
Date: Tue, 17 Jan 2012 16:50:57 +0200
Subject: remove network location provider from default config

android.hardware.location.network is only available if
Network Location Provider is available. Not all configurations
that include gps have NLP, so this should be removed from
default configuration files. For devices that have NLP
android.hardware.location.network.xml should be installed.

Change-Id: I3a25c5687dbae62972fac1bd789a67c81e00d8b8
Original-Change-Id: Iab06f0fff3f109eee24186548f2624e0a97f2104
Signed-off-by: Irina Tirdea <irina.tirdea@intel.com>
---
 data/etc/android.hardware.location.gps.xml |    3 ++-
 data/etc/tablet_core_hardware.xml          |    3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/data/etc/android.hardware.location.gps.xml b/data/etc/android.hardware.location.gps.xml
index 72ab732..d13dd5e 100644
--- a/data/etc/android.hardware.location.gps.xml
+++ b/data/etc/android.hardware.location.gps.xml
@@ -17,6 +17,7 @@
 <!-- These are the location-related features for devices that include GPS. -->
 <permissions>
     <feature name="android.hardware.location" />
-    <feature name="android.hardware.location.network" />
     <feature name="android.hardware.location.gps" />
+    <!-- devices with Network Location Provider  must include
+         android.hardware.location.xml -->
 </permissions>
diff --git a/data/etc/tablet_core_hardware.xml b/data/etc/tablet_core_hardware.xml
index 207fc9a..fbce8fb 100644
--- a/data/etc/tablet_core_hardware.xml
+++ b/data/etc/tablet_core_hardware.xml
@@ -24,7 +24,6 @@
 -->
 <permissions>
     <feature name="android.hardware.location" />
-    <feature name="android.hardware.location.network" />
     <feature name="android.hardware.sensor.compass" />
     <feature name="android.hardware.sensor.accelerometer" />
     <feature name="android.hardware.bluetooth" />
@@ -38,6 +37,8 @@
     <feature name="android.software.home_screen" />
     <feature name="android.software.input_methods" />
     <!-- devices with GPS must include android.hardware.location.gps.xml -->
+    <!-- devices with Network Location Provider  must include
+         android.hardware.location.xml -->
     <!-- devices with a rear-facing camera must include one of these as appropriate:
          android.hardware.camera.xml or 
          android.hardware.camera.autofocus.xml or 
-- 
1.7.4.1

